DolbyIO/Source/Private/DolbyIOSubsystem.cpp

```diff
@@ -351,12 +353,16 @@ void UDolbyIOSubsystem::Initialize(const FString& Token)
 		        return Sdk->conference().add_event_handler(
 		            [this](const dolbyio::comms::video_track_added& Event)
 		            {
-			            if (Event.remote)
+			            if (Event.track.remote)
 			            {
-				            const FString ParticipantID = Event.peer_id.c_str();
-				            const FString StreamID = Event.stream_id.c_str();
+				            const FString ParticipantID = Event.track.peer_id.c_str();
+				            const FString StreamID = Event.track.stream_id.c_str();
 				            DLB_UE_LOG("Video track added: ParticipantID=%s StreamID=%s", *ParticipantID, *StreamID);
-				            VideoSink->AddStream(ParticipantID, StreamID);
+				            VideoSinks.Emplace(ParticipantID, std::make_unique<DolbyIO::FVideoSink>());
```

Comment:
you mean std::make_shared, I guess, but do we need smart pointer here? I would use plain DolbyIO::FVideoSink as value in map.

Reply:
That would mean there are different FVideoSinks in the subsystem and in the SDK. Or am I misunderstanding your intention? I want the same object here and in the SDK and I want it to die only when it is not referenced anymore. shared_ptr is the right choice I believe. TSharedPtr, as you suggested in another comment, could perhaps work, but the SDK expects a shared_ptr and I don't want to mix these two.

Reply:
sorry, I missed that std::shared_ptr is used by sdk - so I suggest only this:
```suggestion
				            VideoSinks.Emplace(ParticipantID, std::make_shared<DolbyIO::FVideoSink>());
```